packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -395,7 +395,7 @@ export class ConnectionFlow {
   }
 
   handleLoginSuccess() {
-    this.jobWatcher.handleLoginSuccess()
+    this.jobWatcher?.handleLoginSuccess()
```

Comment:
how is it possible? How can we want to handle login success without having a jobWatcher?

Reply:
This is a mistake, it cannot happen

Reply:
commit to remove?

Reply:
@doubleface should we remove this commit?

Reply:
done